cluster/restore-from-backup.sh

```diff
@@ -153,16 +153,15 @@ if [ "${ETCD_API}" == "etcd2" ]; then
   mkdir -p "${BACKUP_DIR}/member/snap"
   mkdir -p "${BACKUP_DIR}/member/wal"
   # If the cluster is relatively new, there can be no .snap file.
-  mv *.snap "${BACKUP_DIR}/member/snap/" || true
-  mv *.wal "${BACKUP_DIR}/member/wal/"
+  mv ./*.snap "${BACKUP_DIR}/member/snap/" || true
```

Comment:
`*.snap` could become an argument if you had say `-.snap` on disk. unprefixed globs are bad practice.
the leading `./` shouldn't be considered an argument by anything.